dev-docs/modules/userId.md

```diff
@@ -393,25 +393,25 @@ The `request.userId.lipb` object would then look like
 
 The adapters can be implemented to use the lipibid as the identifier, and segments to which that identifier is associated with.
 
-#### LiveIntent ID Registration
-
 For identity resolution to work for a specific publisher, LiveIntent has to build a model on the backend. In order to collect data for this model an additional call is issued on each page load.
 
-#### Registering your own first party cookie space
+#### LiveIntent ID Registration
+
+To leverage the LiveIntent ID, you need to first set up a first-party cookie sync with LiveIntent. Please reach out to peoplebased@liveintent.com for more information.
```

Comment:
Is this true?  I though the version 2 module would work with out Liveintent tags on page?